includes/pmpro-gift-levels.php

```diff
@@ -3,7 +3,7 @@
  * Load Email Template
  */
 function pmprowoo_gift_levels_email_path($default_templates, $page_name, $type = 'email', $where = 'local', $ext = 'html') {
-  $default_templates[] = PMPROWC_DIR . "/email/{$page_name}.{$ext}";
+  $default_templates[] = esc_url( PMPROWC_DIR . "/email/{$page_name}.{$ext}" );

```

Comment:
This one needs checking, I think esc_url is right here, maybe esc_raw_url?

Reply:
sanitize_file_name
```suggestion
  $default_templates[] = PMPROWC_DIR . '/email/' . sanitize_file_name( $page_name . '.' . $ext );

```